fluidBuild.config.cjs

```diff
@@ -634,6 +634,7 @@ module.exports = {
 			tree: { heading: "🌳 SharedTree DDS changes" },
 			fix: { heading: "🐛 Bug Fixes" },
 			deprecation: { heading: "⚠️ Deprecations" },
+			legacy: { heading: "☠️ Legacy API Changes" },
```

Comment:
That's a skull and crossbones ☠️  Should we use something else?

Reply:
Maybe t-rex? 🦖 
😄 

I'll just remove it. It's helpful as a visual anchor but if it has little reference to the heading, it's more distracting than helpful.